Dockerfile.deploy

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+MAINTAINER "Aslak Knutsen <aslak@redhat.com>"
+ENV LANG=en_US.utf8
+ENV ALMIGHTY_INSTALL_PREFIX=/usr/local/alm
+ENV KUBECONFIG_DIR=${ALMIGHTY_INSTALL_PREFIX}/.kube
+ENV KUBECONFIG=${KUBECONFIG_DIR}/config
+
+# Create a non-root user and a group with the same name: "almighty"
+ENV ALMIGHTY_USER_NAME=almighty
+RUN useradd --no-create-home -s /bin/bash ${ALMIGHTY_USER_NAME}
+
+COPY bin/fabric8-notification ${ALMIGHTY_INSTALL_PREFIX}/bin/fabric8-notification
+
+# From here onwards, any RUN, CMD, or ENTRYPOINT will be run under the following user
+USER ${ALMIGHTY_USER_NAME}
+
+WORKDIR ${ALMIGHTY_INSTALL_PREFIX}
```

Comment:
fixed in https://github.com/fabric8-services/fabric8-notification/pull/20/commits/d422c4ac62f3f26b4a5be498c6086bf117b04669